yarn.lock

```diff
@@ -3830,31 +3830,6 @@
   resolved "https://registry.yarnpkg.com/@protobufjs/utf8/-/utf8-1.1.0.tgz#a777360b5b39a1a2e5106f8e858f2fd2d060c570"
   integrity sha1-p3c2C1s5oaLlEG+OhY8v0tBgxXA=
 
-"@sendgrid/client@^6.5.5":
```

Comment:
Is this yarn.lock meant to be included?

Reply:
dont know why the lock file changed, but ill remove it

Reply:
You should merge master I merged a new `yarn.lock`